src/wallet/rpcwallet.cpp

```diff
@@ -2598,8 +2599,21 @@ static UniValue createwallet(const JSONRPCRequest& request)
         flags |= WALLET_FLAG_DISABLE_PRIVATE_KEYS;
     }
 
+    bool create_blank = false;
     if (!request.params[2].isNull() && request.params[2].get_bool()) {
         flags |= WALLET_FLAG_BLANK_WALLET;
+        create_blank = true;
+    }
+    SecureString passphrase;
+    passphrase.reserve(100);
+    if (!request.params[3].isNull()) {
+        passphrase = request.params[3].get_str().c_str();
+        if (!passphrase.empty()) {
+            if (create_blank) {
+                throw JSONRPCError(RPC_WALLET_ERROR, "Cannot create an encrypted empty wallet.");
+            }
```

Comment:
Add comment why you're temporarily making the wallet blank:

Reply:
Done